specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/2017-06-15-preview/hanaonazure.json

```diff
@@ -0,0 +1,380 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "HanaManagementClient",
+        "description": "The SAP HANA on Azure Management Client.",
+        "version": "2017-06-15-preview"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "securityDefinitions": {
+        "azure_auth": {
+            "type": "oauth2",
+            "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+            "flow": "implicit",
+            "description": "Azure Active Directory OAuth2 Flow",
+            "scopes": {
+                "user_impersonation": "impersonate your user account"
+            }
+        }
+    },
+    "paths": {
+        "/providers/Microsoft.HanaOnAzure/operations": {
+            "get": {
+                "tags": [
+                    "HanaOnAzure"
+                ],
+                "operationId": "HanaOperations_List",
+                "description": "Gets a list of SAP HANA management operations.",
+                "x-ms-examples": {
+                    "List all HANA management operations supported by HANA RP": {
+                        "$ref": "./examples/HanaOperations_List.json"
+                    }
+                },
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
```

Comment:
please add
```
"x-ms-pageable": {
  "nextLinkName": null
}
```
(see other RPs), that will cause the `value` property of `OperationList` to be flattened out, i.e. the method's return type will be a collection rather than an object with one property that is a collection. Better UX 🙂

Reply:
Added